pkg/util/exec/exec.go

```diff
@@ -269,7 +268,7 @@ func runCommandWithOutput(cmd *exec.Cmd, combinedOutput bool) (string, error) {
 	out := strings.TrimSpace(string(output))
 
 	if err != nil {
-		return out, errors.Wrapf(err, "%s", string(err.(*exec.ExitError).Stderr))
+		return fmt.Sprintf("%s. %s", out, string(err.(*exec.ExitError).Stderr)), err
```

Comment:
if combinedOutput is true, the stderr will already be included in the `out` var. The `cmd.CombinedOutput()` call on line 263 will return it.

Reply:
Ok so I should do this only in L265 right?